docs/ModStorage/index.md

```diff
@@ -0,0 +1,91 @@
+# Mod Storage
+
+Read [Mod Structure](../ModStructure/index.md) if you haven't done so already to set up the storage version.
+
+## 
+
+In `3_Game` script module, create the following class.
+
+```csharp
+class NameOfMod : ModStructure
+{
+	override void LoadData()
+	{
+		super.LoadData();
+
+		SetStorageVersion(2);
+	}
+};
+```
```

Comment:
This was already mentioned in docs/cfgmods? I'd think the best place is here, because its used for modstorage. And Then just say what you did in the other md so people know here that either they set the cfgmods config.cpp entry (which should be the first option suggested, as it is the easiest by default) and then the more advanced approach if you want to set it via script.

Remove the mention from cfgmods docs md file, maybe merge this together if the other md has no right to live on its own ... right now you need to look two places to find out you can also set the config var and not make a class for it.